packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -694,13 +694,10 @@ class _Paths {
     if (rxRaw != null && rxRaw != '') {
       final double rx = parserState.parseDoubleWithUnits(rxRaw)!;
       final double ry = parserState.parseDoubleWithUnits(ryRaw)!;
-
-      return (PathBuilder()
-            ..addRRect(Rect.fromLTRB(x, y, w - x, h - y), rx, ry))
-          .toPath();
+      return PathBuilder().addRRect(Rect.fromLTWH(x, y, w, h), rx, ry).toPath();
```

Comment:
main change here is `Rect.fromLTWH` instead of incorrectly calcuating RB from WH.